serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -175,6 +174,16 @@ The previous command results in the following container image:
 ====
 In case GraalVM is not installed, you can set the `quarkus.native.container-build=true` system property, which creates a Linux executable.
 ====
+[IMPORTANT]
+.Configure Docker to use the in-cluster (Remote) Docker daemon
```

Comment:
I think you shouldn't start this sentence with a period (`.`). It's inside an `IMPORTANT` box.